super_team_picker/app.js

```diff
@@ -0,0 +1,55 @@
+const express = require("express")
+const app = express()
+const superTeamRouter = require("./routes/super_team_picker")
+const cookieParser = require('cookie-parser');
+const logger = require("morgan")
+const path = require("path")
+const methodOverride = require("method-override")
+
+app.use(logger('dev'))
+app.use(express.urlencoded({extended: true}))
+
+app.use(methodOverride((req, res) => {
+  if(req.body && req.body._method) {
+    const method = req.body._method
+    return method
+  }
+}))
+
+app.use(cookieParser())
+app.use(express.static(path.join(__dirname, 'public')))
+
+app.use((req, res, next) => {
+    
+    const username = req.cookies.username
+    res.locals.username = ''
+
+    if(username){
+        res.locals.username = username;
+    }   
+    next(); 
+})
+
```

Comment:
You need to define the root route otherwise loading `localhost:3000` errors out with 404. Root route should always be defined even if you don't do anything there.